aptos/token_bridge/sources/bridge_state.move

```diff
@@ -159,10 +303,18 @@ module token_bridge::bridge_state {
         table::upsert(&mut state.bridge_implementations, chain_id, bridge_contract);
     }
 
-    public entry fun set_wrapped_asset(token_chain_id: U16, token_address: vector<u8>, wrapper: vector<u8>) acquires State {
+    public entry fun set_wrapped_asset(native_info: &OriginInfo, wrapper: vector<u8>) acquires State {
+        let state = borrow_global_mut<State>(@token_bridge);
+        let origin_info_to_wrapped_assets = &mut state.origin_info_to_wrapped_assets;
+        table::upsert(origin_info_to_wrapped_assets, *native_info, wrapper);
+        let is_wrapped_asset = &mut state.is_wrapped_asset;
+        table::upsert(is_wrapped_asset, wrapper, true);
+    }
+
+    public entry fun set_native_info(wrapper: vector<u8>, native_info: &OriginInfo) acquires State {
```

Comment:
likewise, I don't think this should be `public` or `entry`

Reply:
made it private